STYLE.md

```diff
@@ -127,12 +127,36 @@ spelled the same.
 
 ## Units
 
-- **Currency:** all amounts should be denominated in BTC.  E.g. "the
-  default minimum relay fee is 0.00000001 BTC/vbyte".  When writing out
-  the name of the currency, pluralize by appending an `s`.  E.g. "Alice
-  owns several bitcoins" not ~~"Alice owns several bitcoin"~~.
-
-- **Weight:** use vbytes rather than weight units, unless using weight
+- **Currency:** acceptable units and abbreviations are:
+
+    | Unit | Abbreviations | Value |
+    |-|-|-|
+    | bitcoins | **BTC** | |
+    | millibitcoins | mBTC | 0.001 BTC |
+    | microbitcoins | µBTC, uBTC | 0.000001 BTC |
+    | satoshis | **sat** | 0.00000001 BTC |
+    | nanobitcoins | nBTC | 0.000000001 BTC |
+    | millisatoshis | msat | 0.00000000001 BTC |
+    | picobitcoins | pBTC | 0.000000000001 BTC |
+
+    The abbreviations bolded above may always be used without
+    introduction; we encourage introduction of other abbreviations, e.g.
+    "you can send 1 millisatoshi (msat) on LN".
+
+    The pluralization and agreement rules for bitcoin/bitcoins,
+    satoshi/satoshis, and derived units are the same as for
+    dollar/dollars (USD).  Similarly, abbreviations are never
+    pluralized.
+
+- **Feerates:** the preferred unit is "sat/vbyte" (satoshis per vbyte).
+  When used only sporadically within a page or section, it is preferred
+  to write out "sat/vbyte" each time, but when used frequently, you may
+  introduce and use the abbreviation "sat/vB".  See note in
+  *weight/vbytes* item about using SI prefixes.
+
+- **Weight/vbytes:** use vbytes rather than weight units, unless using weight
```

Comment:
perhaps introduce the abbreviation here: "weight units (WU), unless..."

Reply:
I think the abbreviation for weight units is unambiguous, so it probably doesn't need to be explicitly mentioned.

Reply:
No strong opinion, but I should have mentioned that my comment was motivated by someone recently asking on irc what "WU" meant.